xbmc/filesystem/VideoDatabaseDirectory/DirectoryNodeMoviesOverview.cpp

```diff
@@ -24,7 +24,8 @@ Node MovieChildren[] = {
                         { NODE_TYPE_STUDIO,       "studios",    20388 },
                         { NODE_TYPE_SETS,         "sets",       20434 },
                         { NODE_TYPE_COUNTRY,      "countries",  20451 },
-                        { NODE_TYPE_TAGS,         "tags",       20459 }
+                        { NODE_TYPE_TAGS,         "tags",       20459 },
+                        { NODE_TYPE_TYPES,        "types",      39300 }
```

Comment:
please add a comma at the end of the line for the same reason I already explained
```suggestion
                        { NODE_TYPE_TYPES,        "types",      39300 },
```

Reply:
Will do. Forgot this.